README.md

```diff
@@ -148,3 +148,25 @@ curl "http://docker.localhost:5080/katsu/api/datasets" \
 curl "http://docker.localhost:3333/htsget/v1/variants/NA20787" \
```

Comment:
Near the top of this file, under `## Authorizing users for the new dataset`, it may be helpful to mention that you should restart the OPA container to get it to work.